src/script/globals/extra.mjs

```diff
@@ -0,0 +1,120 @@
+export const setglobal = () => {
+  return 1;
+};
+
+export const getglobal = () => {
+  return 1;
+};
+
+export const strtrim = () => {
+  return 1;
+};
+
+export const strsplit = () => {
+  return 1;
+};
+
+export const strjoin = () => {
+  return 1;
+};
+
+export const strreplace = () => {
+  return 1;
+};
+
+export const strconcat = () => {
+  return 1;
+};
+
+export const strlenutf8 = () => {
+  return 1;
+};
+
+export const issecure = () => {
+  return 1;
+};
+
+export const issecurevariable = () => {
+  return 1;
+};
+
+export const forceinsecure = () => {
+  return 1;
+};
+
+export const securecall = () => {
+  return 1;
+};
+
+export const hooksecurefunc = () => {
+  return 1;
+};
+
+export const debugload = () => {
+  return 1;
+};
+
+export const debuginfo = () => {
+  return 1;
+};
+
+export const debugprint = () => {
+  return 1;
+};
+
+export const debugdump = () => {
+  return 1;
+};
+
+export const debugbreak = () => {
+  return 1;
+};
+
+export const debughook = () => {
+  return 1;
+};
+
+export const debugtimestamp = () => {
+  return 1;
+};
+
+export const debugprofilestart = () => {
+  return 1;
+};
+
+export const debugprofilestop = () => {
+  return 1;
+};
+
+export const seterrorhandler = () => {
+  // TODO: Implement Lua-side error handler
+  return 0;
+};
+
+export const geterrorhandler = () => {
+  return 1;
+};
+
+export const date = () => {
+  return 1;
+};
+
+export const time = () => {
+  return 1;
+};
+
+export const difftime = () => {
+  return 1;
+};
+
+export const debugstack = () => {
+  return 1;
+};
+
+export const debuglocals = () => {
+  return 1;
+};
+
+export const scrub = () => {
+  return 1;
+};
```

Comment:
If my current understanding is correct, the return value for these Lua interop functions (ie the `return 1`s above) is effectively the number of arguments pushed to the Lua stack. I didn't realize this when originally stubbing functions, hence why I used `return 1`.

Since a stubbed function doesn't push any values to the stack, we may want to default to `return 0` until we implement the function. This should make the Lua side of things consider the return value of the function call in Lua to be `nil`.

Reply:
Ah yes! Made these changes locally, I'll incorporate them in a future PR.

Reply:
Fixed 👍